suricata.yaml.in

```diff
@@ -964,7 +964,15 @@ app-layer:
       #enabled: yes
     ssh:
       enabled: yes
-      #hassh: yes
+      # hassh: no
+
+      # What to do when the encrypted communications start:
+      # - track-only: keep tracking but stop inspection
+      # - full:    keep tracking and inspect as normal
+      # - bypass:  stop processing this flow as much as possible.
+      #            Offload flow bypass to kernel or hardware if possible.
+      #
+      # encryption-handling: track-only
```

Comment:
Should we clearly indicate here and for tls what is the default?
Should we also have the note about performance here, as we have for tls?

Reply:
added the indication. I may not like it so much because now we indicate the default at two different places (the commented out configuration property and added "(default)" to the configuration options but we can consider it in the next PR. It is more user-friendly this way though, my only problem was keeping the same information at two places.